dyn/tm/services/dsf.py

```diff
@@ -2179,7 +2179,7 @@ def _post(self, service_id, publish=True):
         """
         self.service_id = service_id
         uri = '/DSFResponsePool/{}/'.format(self.service_id)
-        api_args = {'publish': 'Y', 'label': self._label,
+        api_args = {'publish': 'N', 'label': self._label,
```

Comment:
I doesn't really matter functionally (in this case),  but you can just exclude the 'publish' :'N' from this line. I'd hazard a guess there are probably more of these in this file.